app/scripts/views/visits/index.js

```diff
@@ -4,12 +4,27 @@
 
 define([
   'views/base',
-  'stache!templates/visits/index'
-], function (BaseView, VisitsIndexTemplate) {
+  'stache!templates/visits/index',
+  'collections/visits',
+  'views/visits/item'
+], function (BaseView, VisitsIndexTemplate, Visits, VisitsItemView) {
   'use strict';
 
   var VisitsIndexView = BaseView.extend({
     template: VisitsIndexTemplate,
+
+    initialize: function () {
+      this.collection = new Visits();
+
+      this.listenTo(this.collection, 'add destroy reset', this.render);
```

Comment:
Hmm it might be a bad idea to trigger render on reset and add. I'll look into this.